ogc-example/src/main/scala/geotrellis/server/ogc/Main.scala

```diff
@@ -118,7 +118,8 @@ object Main extends CommandApp(
               WmsModel(
                 svc.serviceMetadata,
                 svc.parentLayerMeta,
-                svc.layerSources(simpleSources)
+                svc.layerSources(simpleSources),
+                FocalParameters.extendedParametersBinding
```

Comment:
Function that knows how to map over nodes is optional and passed during the service compile time.